tests/test_spheropolygon.py

```diff
@@ -37,28 +37,26 @@ def test_2d_verts(square_points):
 
 
 @given(floats(0.1, 1000))
```

Comment:
Keyword `given`s also help avoid this problem explicitly.
```suggestion
@given(r=floats(0.1, 1000))
```